modules/iam/main.tf

```diff
@@ -1,25 +1,46 @@
+locals {
+  role_name = format(
+    "%s-%s-%s",
+    var.project,
+    var.role_name == "policyname" ? var.policy : var.role_name,
+    var.environment
+  )
+  policy_name = format(
+    "%s-%s-%s",
+    var.project,
+    var.role_name == "policyname" ? var.policy : var.role_name,
+    var.environment
+  )
```

Comment:
policy name and role name are the same. Is that expected?

Reply:
Yes, from original logic.